packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -83,17 +81,17 @@ contract CrabMigration is Ownable {
     event ClaimAndWithdraw(address indexed user, uint256 crabAmount);
 
     modifier beforeMigration() {
-        require(!isMigrated, "M2");
+        require(!isMigrated, "M1");
         _;
     }
 
     modifier afterMigration() {
-        require(isMigrated, "M3");
+        require(isMigrated, "M2");
         _;
     }
 
     modifier afterInitialized() {
-        require(address(crabV2) != address(0), "M8");
+        require(address(crabV2) != address(0), "M7");
```

Comment:
This the wrong error code (and was wrong before).  We need to keep the M10 error code from before.

Reply:
added